tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TO_TAG="$1"
+
+FROM_TAG=$(grep "docker.io/rook/ceph" deploy/examples/images.txt | awk -F : '{ print $2 }')
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SED="${scriptdir}/../../build/sed-in-place"
+
+# This script is supposed to facilitate the preparation of a rook release by
+# changing example and documentation files to use the new release version (TO_TAG)instead the old one (FROM_TAG).
+#
+# It will usually be run on a release branch
+
+FILES_DOCS=(
+ Documentation/Getting-Started/quickstart.md
+ Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md
+ Documentation/Upgrade/rook-upgrade.md
+)
+FILES_CHARTS=(
+ deploy/charts/rook-ceph/values.yaml
+)
+mapfile -t FILES_EXAMPLES < <(find deploy/examples -type f)
+
+$SED "s/ $FROM_TAG / $TO_TAG /g" "${FILES_DOCS[@]}"
+$SED "s/tag: $FROM_TAG/tag: $TO_TAG/g" "${FILES_CHARTS[@]}"
+$SED "s/docker.io\/rook\/ceph:$FROM_TAG/docker.io\/rook\/ceph:$TO_TAG/g" "${FILES_EXAMPLES[@]}"
```

Comment:
```suggestion
$SED "s/docker.io\/rook\/ceph:$FROM_TAG/docker.io\/rook\/ceph:$TO_TAG/g" deploy/examples/*.yaml
$SED "s/docker.io\/rook\/ceph:$FROM_TAG/docker.io\/rook\/ceph:$TO_TAG/g" deploy/examples/images.txt
```

Reply:
Alternately, I think this could be one line which could avoid duplicating the search-replace expression

```
$SED "s/docker.io\/rook\/ceph:$FROM_TAG/docker.io\/rook\/ceph:$TO_TAG/g" deploy/examples/*.yaml deploy/examples/images.txt
```